mypyc/irbuild/classdef.py

```diff
@@ -312,23 +463,41 @@ def setup_non_ext_dict(builder: IRBuilder,
     return non_ext_dict
 
 
+def add_non_ext_class_attr_ann(builder: IRBuilder,
+                               non_ext: NonExtClassInfo,
+                               lvalue: NameExpr,
+                               stmt: AssignmentStmt,
+                               get_type_info: Optional[Callable[[AssignmentStmt],
+                                                                Optional[TypeInfo]]] = None
+                               ) -> None:
+    """Add a class attribute to __annotations__ of a non-extension class.
+    """
```

Comment:
Style nit: Move final `"""` to the same line as the initial `"""`.

Reply:
fixed. I also fixed another preexisting function in this module with this style which led me astray.